src/model/channel/guild_channel.rs

```diff
@@ -102,6 +102,12 @@ pub struct GuildChannel {
     /// channels.
     #[serde(default, rename = "rate_limit_per_user")]
     pub slow_mode_rate: Option<u64>,
+    /// The region override.
+    ///
```

Comment:
Would you be able to clarify here that a `None` value on this field for a compatible channel (voice/stage) means automatic region selection?